examples/hf_ptq/scripts/huggingface_example.sh

```
@@ -297,11 +297,16 @@ if [[ $TASKS =~ "lm_eval" ]]; then
+    LM_EVAL_TP=$(python -c "import torch; print(max(torch.cuda.device_count(), 1))")
 
-    python lm_eval_tensorrt_llm.py \
-        --model trt-llm \
-        --model_args tokenizer=$MODEL_PATH,checkpoint_dir=$SAVE_PATH,max_gen_toks=$BUILD_MAX_OUTPUT_LEN \
+    echo "Using the following config: max output $BUILD_MAX_OUTPUT_LEN max batch $BUILD_MAX_BATCH_SIZE tp $LM_EVAL_TP"
+
+    # max_input_len defaults to 2048, which silently truncates 5-shot prompts; size the
+    # engine for a 4096-token context plus the requested generation length.
+    python lm_eval_hf.py \
+        --model trtllm \
+        --model_args model=$SAVE_PATH,tokenizer=$MODEL_PATH,tensor_parallel_size=$LM_EVAL_TP,max_batch_size=$BUILD_MAX_BATCH_SIZE,max_gen_toks=$BUILD_MAX_OUTPUT_LEN,max_input_len=4096,max_output_len=$BUILD_MAX_OUTPUT_LEN \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)